docs/developer_setup.md

```diff
@@ -27,4 +27,12 @@ docker-compose run --rm web
 
 ## Accessing the Site
 
-If everything worked as designed your site should now be available at [https://boston-cas.dev.test](https://boston-cas.dev.test).  Any mail that the site sends will be delivered to [MailHog](https://github.com/mailhog/MailHog) which is availble at [https://mail.boston-cas.dev.test](https://mail.boston-cas.dev.test)
\ No newline at end of file
+If everything worked as designed your site should now be available at [https://boston-cas.dev.test](https://boston-cas.dev.test).  Any mail that the site sends will be delivered to [MailHog](https://github.com/mailhog/MailHog) which is availble at [https://mail.boston-cas.dev.test](https://mail.boston-cas.dev.test)
+
+## Additional Notes
```

Comment:
[Can you add this here](https://github.com/greenriver/hmis-warehouse/blob/stable/docs/developer_setup.md) too?

Reply:
Just added a new PR for the warehouse version of this document